server/infinispan/src/test/java/org/hawkular/apm/server/infinispan/InfinispanSpanCacheTest.java

```diff
@@ -36,13 +36,11 @@
  */
 public class InfinispanSpanCacheTest extends AbstractInfinispanTest {
 
-    private static final String TENANT = "tenant";
-
     private InfinispanSpanCache spanCache;
 
     @Before
     public void before() {
-        spanCache = new InfinispanSpanCache(cacheManager.getCache(InfinispanCommunicationDetailsCache.CACHE_NAME));
+        spanCache = new InfinispanSpanCache(cacheManager.getCache("spans"), cacheManager.getCache("trace"));
```

Comment:
thought in the config the new cache was spanTrace ? and the span cache was 'span'?